linera-execution/src/runtime.rs

```diff
@@ -483,6 +557,12 @@ where
     }
 
     async fn write_batch_and_unlock(&self, batch: Batch) -> Result<(), ExecutionError> {
+        let size = batch.size() as u64;
+        self.bytes_written.fetch_add(size, Ordering::Relaxed);
```

Comment:
(In theory this could overflow. Probably not in practice… but maybe the default for these limits should not be `MAX` but rather `MAX / 2` or something?)